starport/services/scaffolder/init.go

```diff
@@ -51,10 +62,16 @@ func (s *Scaffolder) Init(tracer *placeholder.Tracer, name string, noDefaultModu
 	}
 
 	// initialize git repository and perform the first commit
-	if err := initGit(pathInfo.Root); err != nil {
+	if err := initGit(absRoot); err != nil {
+		return "", err
+	}
+
+	// get the relative app path from the current directory
+	relativePath, err := filepath.Rel(pwd, appPath)
```

Comment:
why don't we return the abs as it's?

Reply:
for we can show a better UI message after scaffolding a chain:

![Screen Shot 2021-09-07 at 12 42 19](https://user-images.githubusercontent.com/2406457/132373430-66eb1275-08d6-41ad-8f53-dfe12a60a5d4.png)